CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+#  oo          MM'"""""`MMM                   dP                d8888b.
+#              M  .mmm,  MM                   88                    `88
+#  dP .d8888b. M  MMMMM  MM dP    dP .d8888b. 88  .dP  .d8888b.  aaad8'
+#  88 88'  `88 M  MM  M  MM 88    88 88'  `88 88888"   88ooood8     `88
+#  88 88.  .88 M  `MM    MM 88.  .88 88.  .88 88  `8b. 88.  ...     .88
+#  dP `88888P' MM.    .. `M `88888P' `88888P8 dP   `YP `88888P' d88888P
+#              MMMMMMMMMMMM
+
+cmake_minimum_required(VERSION 3.5)
+project(Tremulous C CXX)
+
+set(CMAKE_CXX_FLAGS -std=c11)
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+endif()
+
+option(USE_RENDERER_DLOPEN "" ON)
+
+if (APPLE OR WIN32)
```

Comment:
AFAIK when building the packaged release for OSX and Windows, you want to use the internal SDL2, adding automagic detection in this case I believe is undesired.